playbook-website/app/views/pages/visual_guidelines_partials/_shadow.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= render :partial => "pages/visual_guidelines_partials/pb_doc_shadow",
+locals:{ title: "Shadow Options",
+colors: [{name:"Shadow None",variable:"shadow_none"},
```

Comment:
I'd like to move these settings to the controller. See this example: 

https://github.com/powerhome/playbook/blob/master/playbook-website/app/controllers/pages_controller.rb#L85